src/vos/vos_common.c

```diff
@@ -479,6 +479,28 @@ vos_tls_init(int xs_id, int tgt_id)
 		D_WARN("Failed to create committed cnt sensor: "DF_RC"\n",
 		       DP_RC(rc));
 
+	rc = d_tm_add_metric(&tls->vtl_dtx_cmt_ent_cnt, D_TM_COUNTER,
+			     "Number of committed entries", "entry",
+			     "mem/vos/dtx_cmt_ent_%u/tgt_%u",
+			     sizeof(struct vos_dtx_cmt_ent), tgt_id);
+	if (rc)
+		D_WARN("Failed to create committed cnt: "DF_RC"\n",
+		       DP_RC(rc));
+
+	rc = d_tm_add_metric(&tls->vtl_obj_cnt, D_TM_COUNTER,
+			     "Number of cached vos object", "entry",
+			     "mem/vos/vos_obj_%u/tgt_%u",
+			     sizeof(struct vos_object), tgt_id);
+	if (rc)
+		D_WARN("Failed to create vos obj cnt: "DF_RC"\n", DP_RC(rc));
+
+	rc = d_tm_add_metric(&tls->vtl_lru_alloc_size, D_TM_COUNTER,
+			     "Number of LRU cache size", "byte",
```

Comment:
If you were to change the above, this could change to be more specific.  It would be more accurate to say something like "active DTX table LRU cache".

Reply:
sure